### ISSUE TITLE
Fail to get cluster from orchestrator: [orc]: status: 500 msg: at error, json unmarshal error:

### ISSUE THREAD

**Issue by HBO2** [presslabs/mysql-operator#153]
HI @AMecea Please find below a snippet form de operator log:

```
E1116 10:17:45.075107       1 orc_reconciliation.go:57] Fail to get cluster from orchestrator: [orc]: status: 500 msg: at error, json unmarshal error: unexpected end of JSON input, details: &lt;nil&gt;. Now tries to register nodes.
I1116 10:17:48.713580       1 orc_reconciliation.go:40] Orchestrator reconciliation for cluster 'ain-production-cluster' started...
W1116 10:17:48.842026       1 orc_reconciliation.go:318] [getRecoveriesToAck]: Cluster is not ready for ack.
I1116 10:17:48.874770       1 utils.go:173] Endpoints for service 'ain-production-cluster-mysql' were up-to-date.
I1116 10:17:50.084302       1 orc_reconciliation.go:40] Orchestrator reconciliation for cluster 'dev-cluster' started...
E1116 10:17:50.093512       1 orc_reconciliation.go:57] Fail to get cluster from orchestrator: [orc]: status: 500 msg: at error, json unmarshal error: unexpected end of JSON input, details: &lt;nil&gt;. Now tries to register nodes.
I1116 10:17:53.882722       1 orc_reconciliation.go:40] Orchestrator reconciliation for cluster 'ain-production-cluster' started...
W1116 10:17:54.029847       1 orc_reconciliation.go:318] [getRecoveriesToAck]: Cluster is not ready for ack.

```

This message is repeating over and over

Any idea's
thx

**Comment by AMecea** [comment]
Hi @HBO2 , what version of Kubernetes do you use?

**Comment by HBO2** [comment]
Hi @AMecea 

Azure AKS 11.3

**Comment by AMecea** [comment]
The version 0.1.13 of the operator does not work ok kubernetes 1.11.3, we are almost ready with releasing v0.2.0 which will work on k8s 1.11.x and will solve this issue.

The problem with k8s 1.11 is because of the subresources that are enabled by default and the cluster status is marked as subresource so it won't get updates. If the status is not updated the nodes does not register into orchestrator.

**Comment by AMecea** [comment]
:smile: Hopefully the version 0.2.0 will be released this week.

**Comment by AMecea** [comment]
fixed by #102
